tests/migration_test.go

```diff
@@ -216,6 +224,34 @@ var _ = Describe("Migrations", func() {
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 
 			})
+			It("should be successfully with a cloud init", func() {
+				// Start the VirtualMachineInstance with the PVC attached
+				vmi := tests.NewRandomVMIWithPVC(pvName)
+				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\necho 'hello'\n")
+
+				vmi = runVMIAndExpectLaunch(vmi, 180)
+
+				By("Checking that the VirtualMachineInstance console has expected output")
+				expecter, err := tests.LoggedInAlpineExpecter(vmi)
```

Comment:
can we change this to cirros since it exercises cloud-init?

Reply:
That's a bit tricky as we've built the cdi-http-import-server to only expose the alpine image in the target. So, I'll need to change that and everything that uses this pv to login to cirros.

Thing is, that we are only testing here that the disk is present and not if it's used by the guest OS...

Reply:
ah, don't worry about it then.

Reply:
I saw it too late :)
I've updated the cdi-http-import-server to contain both alpine and cirros.
Had a strange problem with the LoggedInCirrosExpecter.
For some weird reason (in the tests) when the vm was started from a pv, it's hostname in the guest was set to `cirros`, and the expected failed as it was looking for a hostname + " login".
But when it was started from an image, everything was fine, the guest hostname was correct.
I couldn't figure it out and just set (in the test) the hostname to "cirros"... :/